backend/graphql/index.ts

```diff
@@ -86,6 +88,7 @@ const graphQLMiddlewares = {
     duplicateTest: authorizedByAdmin(),
     unarchiveTest: authorizedByAdmin(),
     archiveTest: authorizedByAdmin(),
+    createStudent: authorizedByAdmin(),
```

Comment:
I think this should actually be authorized by teachers?

Reply:
Changed!